.github/workflows/release.yml

```diff
@@ -20,3 +20,6 @@ jobs:
       with:
         version: latest
         args: release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we need to define custom write permissions?
See https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token.